ansible/configure/roles/database-import/tasks/main.yml

```diff
@@ -30,13 +38,20 @@
 - name: Create new database
   command: createdb vmdb_production
 
+- name: Create role 'cloudforms'
+  command: createuser cloudforms
```

Comment:
so do you want me to remove this? It seems innocuous as it's a formality while loading the dump, but somewhere down the line while testing UI, a db query might not proceed as expected if this isn't handled during pg_restore?